Api/Services/HelloService.cs

```diff
@@ -29,40 +30,13 @@ public HelloService(IHttpMessageHandlerFactory factory, ILogger<HelloService> lo
 
         public async Task<string> SayHello(string firstName)
```

Comment:
I suggest we leave both the regular call with all exception handling (not to confuse less experienced developers) and also to add additional method (with different name e.g. SayHelloHelper) to show how it could work with a helper and add some comments on that.